core/drivers/crypto/hisilicon/include/hisi_qm.h

```diff
@@ -147,17 +147,22 @@ struct hisi_qp {
 	enum hisi_drv_status (*parse_sqe)(void *sqe, void *msg);
 };
 
+struct qm_xqc {
+	struct qm_sqc *sqc;
+	struct qm_cqc *cqc;
+	paddr_t sqc_dma;
+	paddr_t cqc_dma;
+};
+
 struct hisi_qm {
 	enum qm_fun_type fun_type;
 	vaddr_t io_base;
 	uint32_t io_size;
 	uint32_t vfs_num;
 	uint32_t version;
 
-	struct qm_sqc *sqc;
-	struct qm_cqc *cqc;
-	paddr_t sqc_dma;
-	paddr_t cqc_dma;
+	struct qm_xqc xqc;
+	struct qm_xqc rsv_xqc;
```

Comment:
I asked before what "rsv" in `rsv_xqc` stands for but didn't get an answer. It seems it's used for configuration only, so how about renaming it to `cfg_xqc`?

Reply:
rsv has no actual meaning.changed to cfg_xqc